docs/copied-from-beats/repositories.asciidoc

```diff
@@ -0,0 +1,140 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/setup-repositories.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
```

Comment:
Could you remove the comments, as they don't seem valid here.

Reply:
I'd like to, but I also want to keep these files like they are in the beats repo so we can copy them over without having to manually merge our changes, if they updated in the beats repo.